lib/Cleantalk/ApbctWP/Cron.php

```diff
@@ -11,9 +11,7 @@ class Cron extends \Cleantalk\Common\Cron
      */
     public function getCronLastStart()
     {
-        global $apbct;
-
-        return $apbct->stats['cron']['last_start'];
+        return get_option('cleantalk_cron_last_start');
```

Comment:
Why using get_option() instead of $apbct?

Reply:
Saving single int option by update_option() is faster than $apbct->save('stats'). So the option store in the single item now. It available to get by get_option() only.